app/code/AKisilenko/ModuleLesson6/Api/AskQuestionRepositoryInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace AKisilenko\ModuleLesson6\Api;
+
+use AKisilenko\ModuleLesson6\Api\Data\AskQuestionInterface;
+use Magento\Framework\Api\SearchCriteriaInterface;
+
+/**
+ * Interface AskQuestionRepositoryInterface
+ * @package AKisilenko\ModuleLesson6\Api
+ */
+interface AskQuestionRepositoryInterface
+{
+    /**
+     * @param Data\AskQuestionInterface $askQuestion
+     * @return mixed
+     */
+    public function save(AskQuestionInterface $askQuestion);
+
+    /**
+     * @param $askQuestionId
+     * @return mixed
```

Comment:
```
 @return mixed 
```
- this is incorrect annotation. Magento API uses these annotations to perform requests data formating, so please check carefully the annotations for all input and output variables of API classes. Didn't worked for me.

Reply:
the correct return type for is getById (please check the implementation as well)
```
* @return \AKisilenko\ModuleLesson6\Api\Data\AskQuestionInterface
```

Reply:
Also input type hinting need to be specified correctly 
```
    /**
     * @param int $askQuestionId
     * @return \AKisilenko\ModuleLesson6\Api\Data\AskQuestionInterface
     */
    public function getById(int $askQuestionId): AskQuestionInterface;

```